src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$dayOfWeek:
+  "mon" 1, "tue" 2, "wen" 3, "thu" 4, "fri" 5, "sat" 6, "sun" 7;
```

Comment:
I heard that the each value should be in a new line, but i'm not sure is it necessarily or not.

Reply:
very likely